src/main/java/com/netcracker/study/objects/borders/Shootable.java

```diff
@@ -0,0 +1,8 @@
+package com.netcracker.study.objects.borders;
+
+public interface Shootable {
+
+    public void onShoot();
+
+    public boolean doesStopBullet();
```

Comment:
isAlive()? may be we need another LifeSupported interface which support such operations: isAlive(), getLiveCount() and etc

Reply:
stopping bullets does not directly depend on whether object is alive, it might be alive and don't stop bullets